how-to-docs/ingress.md

```diff
@@ -10,27 +11,34 @@ The annotation ```kubernetes.io/ingress.class: "nginx-internal"``` is used to sp
 
 In the following example the terms "myapp" and "myproject" have been used, these will need to be changed to the relevant names for your project. Where internal is used, this can be changed for an external ingress - everything else stays the same.
 
+```shell
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    # used to select which on the ingress this resource should be configured on
+	kubernetes.io/ingress.class: "nginx-internal"
+	# indicate the ingress SHOULD speak TLS between itself and pods
+	ingress.kubernetes.io/secure-backends: "true"
+  name: myapp-server-internal
+spec:
+  rules:
+  - host: "myapp.myproject.homeoffice.gov.uk"
+    http:
+      paths:
+      - backend:
+          serviceName: myapp
+          servicePort: 8000
+        path: /
+  tls:
+  - hosts:
+    - "myapp.myproject.homeoffice.gov.uk"
+    # the name of the kubernetes secret in your namespace with tls.crt and tls.key
+    secretName: myapp-github-internal-tls
+```
 
+> Always ensure you are using TLS between the ingress controller and your pods by placing the annotation: "ingress.kubernetes.io/secure-backends": "true". At the moment; this is not enforce though there are plans to enforce by policy.
 
-  
- ```---
- apiVersion: extensions/v1beta1
- kind: Ingress
- metadata:
-   annotations:
-     kubernetes.io/ingress.class: "nginx-internal"
-   name: myapp-server-internal
- spec:
-   rules:
-     - host:"myapp.myproject.homeoffice.gov.uk" 
-       http:
-         paths:
-           - backend:
-               serviceName: myapp
-               servicePort: 8000
-             path: /
-   tls:
-     - hosts:
-         - "myapp.myproject.homeoffice.gov.uk"
-       secretName: myapp-github-internal-tls
-   ```
+#### **Using LetsEncrypt with Ingress**
+
+Assuming you are not bringing your own certificates, LetsEncrypt can be used to acquire certificates for both internal *(behind vpn NOT cluster TLS certs)* and external certificates. Simply place the annotation ```stable.k8s.psg.io/kcm.class: default``` into the ingress resource; A full list of the supported features can be found at [here](https://github.com/PalmStoneGames/kube-cert-manager/blob/master/docs/ingress.md). Note at present we ONLY allow you to request certificates via the ingress resource, not by the third part resource.
```

Comment:
`found at [here]` -> ` found [here]`

`part` -> `party`

Reply:
its a label, not a annotation:

```metadata:
  labels:
    stable.k8s.psg.io/kcm.class: default```